ydb/docs/en/core/maintenance/manual/backup-collections.md

```diff
@@ -0,0 +1,311 @@
+
+View backups through the database schema:
+
+```bash
+# List all collections
+ydb scheme ls .backups/collections/
+
+# View specific collection structure
+ydb scheme ls .backups/collections/shop_backups/
+
+# Check backup timestamps
+ydb scheme ls .backups/collections/shop_backups/ | sort
+```
+
+### Monitor backup chains {#monitor-backup-chains}
+
+Verify backup chain integrity:
+
+```bash
+# List backups in chronological order
+ydb scheme ls .backups/collections/shop_backups/ | sort
+
+# Check individual backup contents
+ydb scheme ls .backups/collections/shop_backups/20250821141519Z_full
+```
+
+## Managing backup collections {#managing-collections}
+
+### Collection information {#collection-information}
+
+Since collections are managed through SQL, browse them using schema commands:
+
+```bash
+# Browse all collections
+ydb scheme ls .backups/collections/
+
+# View collection directory contents
+ydb scheme describe .backups/collections/shop_backups/
+```
+
+### Collection status monitoring {#collection-status-monitoring}
+
+Check the health and status of your collections:
+
+1. **Verify recent backups exist**.
+2. **Check backup chain completeness**.
+3. **Monitor storage usage**.
+4. **Validate backup accessibility**.
+
+## Retention and cleanup {#retention-cleanup}
+
+### Backup chain considerations {#backup-chain-considerations}
+
+Before deleting backups, understand chain dependencies:
+
+- **Full backups**: Required for all subsequent incrementals.
+- **Incremental backups**: Depend on the latest full backup and all subsequent incrementals in the chain.
+- **Chain breaks**: Deleting intermediate backups breaks the chain.
+
+{% note warning %}
+
+Currently, YDB does not provide built-in commands to verify chain dependencies or check chain integrity. You must manually track which backups belong to which chain by:
+- Listing backups chronologically using `ydb scheme ls .backups/collections/<collection_name>/ | sort`
+- Identifying backup types by directory name suffix (`_full` or `_incremental`)
+- Ensuring you delete complete chains (all backups from a full backup through its incrementals)
+
+{% endnote %}
+
+### Manual cleanup strategies {#manual-cleanup-strategies}
+
+**Safe cleanup approach:**
+
+1. Create new full backup.
+2. Verify new backup is complete.
+3. Delete old backup chains (full backup + all its incrementals).
+4. Never delete partial chains.
+
+**Example cleanup workflow:**
+
+```bash
+# 1. Create new full backup
+ydb yql -s "BACKUP \`shop_backups\`;"
+
+# 2. Wait for completion and verify
+ydb operation list incbackup
+
+# 3. Browse backup structure
+ydb scheme ls .backups/collections/shop_backups/ | sort
+
+# 4. Remove old backup directories (entire chains only)
+ydb scheme rmdir -r .backups/collections/shop_backups/20250208141425Z_full/
+ydb scheme rmdir -r .backups/collections/shop_backups/20250209141519Z_incremental/
+ydb scheme rmdir -r .backups/collections/shop_backups/20250210141612Z_incremental/
```

Comment:
Explain how to understand that these incrementals belong to `20250208141425Z_full`? Just because they are following each other chronologically?